sass/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,9 @@
+.breadcrumbs li:after {
```

Comment:
@brendansudol Is this an appropriate time to create a more traditionally named class to encapsulate some styles or would it be better to make small utility classes to do each of the different aspects?